### PR TITLE
(test) O3-2045: Add custom schema e2e test

### PR DESCRIPTION
**Pull request by anjula-sack**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [x] My work includes tests or is validated by existing tests.

## Summary

This test will test for the form creation using a custom schema. As per the discussion, I update the redirection to the edit page after the form being created to improve the UX and also as a solution for the form deletion issue. 

## Screenshots

Screenshot 2023-05-09 at 10 48 36 AM

## Related Issue

https://issues.openmrs.org/browse/O3-2045

### CODE REVIEW

File: e2e/specs/customSchemaForm.spec.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import { FromBuilderPage } from "../pages";
+import customSchema from "../support/customSchema.json";
+
+test("Should be able to create a form using custom schema", async ({
+  page,
+}) => {
+  const fromBuilderPage = new FromBuilderPage(page);
+  await fromBuilderPage.gotoFormBuilder();
+
+  await page.getByRole("button", { name: "Create a new form" }).click();
```
Comment (jayasanka-sack) [inline e2e/specs/customSchemaForm.spec.ts:11]: shall we move these elements to the page? 

ex: 

https://github.com/openmrs/openmrs-esm-patient-management/blob/d648f8110573591e605c1b9a4342f648ca54d5ee/e2e/pages/registrationAndEditPage.ts#L25

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]
Thanks for the PR @anjula-sack !

File: e2e/specs/customSchemaForm.spec.ts (hunk 1 above)
Comment (anjula-sack) [reply e2e/specs/customSchemaForm.spec.ts:11]: sure @jayasanka-sack

Comment (jayasanka-sack) [reply e2e/specs/customSchemaForm.spec.ts]: This comment is still valid @anjula-sack

Comment (anjula-sack) [reply e2e/specs/customSchemaForm.spec.ts]: it is already moved @jayasanka-sack

File: e2e/pages/formBuilderPage.ts [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
-export class HomePage {
+export class FormBuilderPage {
   constructor(readonly page: Page) {}
 
   async gotoFormBuilder() {
     await this.page.goto("form-builder");
   }
+
+  async clickCreateNewFormButton() {
+    await this.page.getByRole("button", { name: "Create a new form" }).click();
```
[the remark this answers is not in the document]
Comment (jayasanka-sack) [reply e2e/pages/formBuilderPage.ts:11]: No, I meant for selectors. Ex:  `this.page.createNewButton.click();`  

https://github.com/openmrs/openmrs-esm-patient-management/blob/d648f8110573591e605c1b9a4342f648ca54d5ee/e2e/pages/registrationAndEditPage.ts#L25

**Pull request merged**